src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (num1) => {
+    counter++;
+
+    return (num2) => {
+      const sum = num1 + num2;
+
+      return counter > 3 && counter % 2 === 0 ? 'Bzzz... Error!' : sum;
```

Comment:
great Job!
maybe its better to separate ternary operator on multiple lines for easy reading.

Reply:
yes it is